inloop/tasks/views.py

```diff
@@ -26,7 +26,7 @@
 def index(request):
     categories = [
         (category, category.completion_info(request.user))
-        for category in Category.objects.order_by("name")
+        for category in Category.objects.order_by("priority")
```

Comment:
Just a question: what happens if two or more priorities are set to the same value (e.g., the default value is 0)? I think there should be a fallback ordering, maybe based on `name` as before (i.e., `Category.objects.order_by("priority, name")`).